src/extension.ts

```diff
@@ -51,7 +60,7 @@ function spotifyAuthentication(spotify: typeof SpotifyWebApi) {
 	
 	const app = express();
 	let server = null;
-	app.get(PATH, (req, res) => {
+	app.get(PATH, (req, res, next) => {
```

Comment:
Don't think this variable is used anymore

```suggestion
	app.get(PATH, (req, res) => {
```